core/divesitehelpers.cpp

```diff
@@ -22,24 +22,20 @@
 #include <QTimer>
 #include <memory>
 
-void reverseGeoLookup(degrees_t latitude, degrees_t longitude, taxonomy_data *taxonomy)
+/** Performs a REST request to a service returning a JSON object. */
+QJsonObject doAsyncRest(QString url, int msTimeout)
 {
-	// By making the QNetworkAccessManager static and local to this function,
-	// only one manager exists for all geo-lookups and it is only initialized
-	// on first call to this function.
```

Comment:
Did this comment become meaningless? If not, please keep it.

Reply:
Sorry, did not mean to drop this comment. Copy-pasta failure.